lms/static/scripts/frontend_apps/components/test/LMSFilePicker-test.js

```diff
@@ -249,7 +238,6 @@ describe('LMSFilePicker', () => {
     // Make successful authorization attempt and wait for the auth window to close.
     fakeApiCall.reset();
     fakeApiCall.resolves([0]);
-    await fakeApiCall;
     wrapper.update();
```

Comment:
This `wrapper.update()` should also be redundant.

Reply:
Fixed!